expression/builtin_string.go

```diff
@@ -575,6 +582,7 @@ func (b *builtinRepeatSig) evalString(row chunk.Row) (d string, isNull bool, err
 	if isNull || err != nil {
 		return "", isNull, errors.Trace(err)
 	}
+	byteLength := len(str)
```

Comment:
It seems that this is rune length, not byte length?
I think we need to add some test cased for multi-byte characters, like Chinese character.

Reply:
Len of String will return the byte length,  you can check it in  https://golang.org/pkg/builtin/#len .
I have added some test of Chinese character.